CODE_OF_CONDUCT.md

```diff
@@ -9,12 +9,11 @@ Specifically:
 * Be kind. Be courteous. Be welcoming.
 * Listen. Consider and acknowledge people's points before responding.
 
-Should you experience anything that makes you feel unwelcome in Flutter's
-community, please contact [conduct@flutter.dev](mailto:conduct@flutter.dev)
-or, if you prefer, directly contact someone on the team, for instance
-[Hixie](mailto:ian@hixie.ch) or [Tim](mailto:timsneath@google.com). We will
-not tolerate harassment from anyone in Flutter's community, even outside
-of Flutter's public communication channels.
+As everyone always starts from somewhere, it is imperative that our community remain an open and diverse group of developers of all varieties. Mistakes are a part of life and will be made, that is how we as humans learn. Mistakes made by less-experienced developers will not be met with negative comments or attitudes, and if so, action will be taken against those who committing them.
```

Comment:
If I may comment, I think `those are committing them` is grammatically incorrect. It can be `who are committing them`, or `who commit them`